indico/modules/rb_new/client/js/common/timeline/MonthlyTimelineContent.jsx

```diff
@@ -0,0 +1,108 @@
+/* This file is part of Indico.
+ * Copyright (C) 2002 - 2018 European Organization for Nuclear Research (CERN).
+ *
+ * Indico is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Indico is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Indico; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+import _ from 'lodash';
+import React from 'react';
+import {toMoment} from 'indico/utils/date';
+import WeeklyTimelineContent, {TimelineRowLabel} from './DailyTimelineContent';
+
+/* eslint-disable no-unused-vars */
+import baseStyle from './TimelineContent.module.scss';
+import style from './WeeklyTimelineContent.module.scss';
+/* eslint-enable no-unused-vars */
```

Comment:
nice, i didn't know you can toggle a rule for a range of lines that's not a block